### PR TITLE
Update TON repositories

### PR DESCRIPTION
**Pull request by tsivarev**

### CODE REVIEW

File: data/ecosystems/t/ton.toml [hunk 1]
```diff
@@ -822,35 +922,62 @@ url = "https://github.com/DavidLee9291/ton-testnet.github.io"
 [[repo]]
 url = "https://github.com/DavidLee9291/tonHack"
 
+[[repo]]
+url = "https://github.com/dedust-io/DefiLlama-Adapters"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:926]: this doesn't look like TON? let's remove it maybe

File: data/ecosystems/t/ton.toml [hunk 2]
```diff
@@ -1359,9 +1616,15 @@ url = "https://github.com/howardpen9/jetton-implementation-in-tact"
 [[repo]]
 url = "https://github.com/howardpen9/lending"
 
+[[repo]]
+url = "https://github.com/howardpen9/lending_record_in_tact"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:1620]: looks like an empty repo, can remove

File: data/ecosystems/t/ton.toml [hunk 3]
```diff
@@ -822,35 +922,62 @@ url = "https://github.com/DavidLee9291/ton-testnet.github.io"
 [[repo]]
 url = "https://github.com/DavidLee9291/tonHack"
 
+[[repo]]
+url = "https://github.com/dedust-io/DefiLlama-Adapters"
+
+[[repo]]
+url = "https://github.com/dedust-io/defillama-server"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:929]: this doesn't look like TON? let's remove it maybe

**Review by eherrerosj — CHANGES_REQUESTED** [review]
super nice additions, just left a couple comments if you can help me out

File: data/ecosystems/t/ton.toml (hunk 3 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml]: Done

File: data/ecosystems/t/ton.toml (hunk 2 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml]: Done

File: data/ecosystems/t/ton.toml (hunk 1 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml]: Done

File: data/ecosystems/t/ton.toml [hunk 4]
```diff
@@ -1029,170 +1275,385 @@ url = "https://github.com/dTelecom/ton"
...
+url = "https://github.com/godemodegame/TonCells"
+
+[[repo]]
+url = "https://github.com/godemodegame/TonSmartChallenge-1"
+
+[[repo]]
+url = "https://github.com/godemodegame/tonswap-web"
+
+[[repo]]
+url = "https://github.com/goplus-labs/BC-Analyzer"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:1596]: can remove

File: data/ecosystems/t/ton.toml [hunk 5]
```diff
@@ -627,6 +763,9 @@ url = "https://github.com/Blockchair/ton-indexer"
 [[repo]]
 url = "https://github.com/boltoshi-labs/bolt"
 
+[[repo]]
+url = "https://github.com/boltoshi-labs/huebel.art"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:767]: how's this using ton?

File: data/ecosystems/t/ton.toml [hunk 6]
```diff
@@ -1029,170 +1275,385 @@ url = "https://github.com/dTelecom/ton"
...
-url = "https://github.com/evaafi/terms_of_use"
+url = "https://github.com/EmelyanenkoK/nominator_pool"
 
 [[repo]]
-url = "https://github.com/evaafi/tonconnect-config"
+url = "https://github.com/EmelyanenkoK/OracleHub"
 
 [[repo]]
-url = "https://github.com/FairJournal/backend"
+url = "https://github.com/EmelyanenkoK/orion-exchange"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:1312]: can probably remove given it's more of an oracle / exchange?

File: data/ecosystems/t/ton.toml [hunk 7]
```diff
@@ -1029,170 +1275,385 @@ url = "https://github.com/dTelecom/ton"
...
-url = "https://github.com/evaafi/telegram-info-bot"
+url = "https://github.com/EmelyanenkoK/nomination-contract"
 
 [[repo]]
-url = "https://github.com/evaafi/terms_of_use"
+url = "https://github.com/EmelyanenkoK/nominator_pool"
 
 [[repo]]
-url = "https://github.com/evaafi/tonconnect-config"
+url = "https://github.com/EmelyanenkoK/OracleHub"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:1309]: can probably remove given it's more of an oracle / exchange?

File: data/ecosystems/t/ton.toml [hunk 8]
```diff
@@ -1029,170 +1275,385 @@ url = "https://github.com/dTelecom/ton"
...
-url = "https://github.com/FairJournal/frontend"
+url = "https://github.com/EmelyanenkoK/pyTON"
 
 [[repo]]
-url = "https://github.com/Fantosi/nftchef-monorepo"
+url = "https://github.com/EmelyanenkoK/pytonv3"
 
 [[repo]]
-url = "https://github.com/fck-foundation/.github"
+url = "https://github.com/EmelyanenkoK/rlottie.github.io"
```
Comment (eherrerosj) [inline data/ecosystems/t/ton.toml:1324]: can probably remove

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a few comments

File: data/ecosystems/t/ton.toml (hunk 5 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:767]: It's a landing page for a TON-based token https://huebel.art/

File: data/ecosystems/t/ton.toml (hunk 6 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:1312]: Done

File: data/ecosystems/t/ton.toml (hunk 7 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:1309]: Done. Thank you!

File: data/ecosystems/t/ton.toml (hunk 8 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:1324]: Done. Thank you!

File: data/ecosystems/t/ton.toml [hunk 9]
```diff
@@ -1029,170 +1275,385 @@ url = "https://github.com/dTelecom/ton"
...
+url = "https://github.com/godemodegame/tonswap-web"
+
+[[repo]]
+url = "https://github.com/goplus-labs/BC-Analyzer"
+
+[[repo]]
+url = "https://github.com/goplus-labs/langchain-tutorials"
+
+[[repo]]
+url = "https://github.com/goplus-labs/solidity"
```
[the remark this answers is not in the document]
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:1602]: Removed

File: data/ecosystems/t/ton.toml [hunk 10]
```diff
@@ -8,38 +8,48 @@ github_organizations = [
...
   "https://github.com/Connecton",
   "https://github.com/continuation-team",
   "https://github.com/copperbits",
+  "https://github.com/coursefactory",
   "https://github.com/DAOTON-io",
+  "https://github.com/dedust-io",
   "https://github.com/delab-team",
+  "https://github.com/DEXTON-ton",
   "https://github.com/disintar",
+  "https://github.com/eqtlab",
```
[the remark this answers is not in the document]
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:28]: Updated

File: data/ecosystems/t/ton.toml [hunk 11]
```diff
@@ -8,38 +8,48 @@ github_organizations = [
...
   "https://github.com/evaafi",
   "https://github.com/FairJournal",
   "https://github.com/fck-foundation",
+  "https://github.com/formony",
   "https://github.com/getgems-community-tact",
   "https://github.com/getgems-io",
+  "https://github.com/goplus-labs",
```
[the remark this answers is not in the document]
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:35]: Done

File: data/ecosystems/t/ton.toml [hunk 12]
```diff
@@ -8,38 +8,48 @@ github_organizations = [
   "https://github.com/0xNebulae",
   "https://github.com/1ton-labs",
   "https://github.com/agora-ai",
+  "https://github.com/AllJointTW",
```
[the remark this answers is not in the document]
Comment (tsivarev) [reply data/ecosystems/t/ton.toml:11]: Left only repo

File: data/ecosystems/t/ton.toml (hunk 4 above)
Comment (tsivarev) [reply data/ecosystems/t/ton.toml]: Done

File: data/ecosystems/t/ton.toml [hunk 13]
```diff
@@ -1702,181 +2292,349 @@ url = "https://github.com/klim0v/tact"
...
-url = "https://github.com/Libgram/libgram-front-smofast-smmpanel"
+url = "https://github.com/markokhman/-tonhub-price-history-extension"
 
 [[repo]]
-url = "https://github.com/Libgram/NFT-Landing-TON-Dog-Metaverse"
+url = "https://github.com/markokhman/first_contract_front_end"
 
 [[repo]]
-url = "https://github.com/Libgram/noscript"
+url = "https://github.com/markokhman/Font-Awesome"
```
[the remark this answers is not in the document]
Comment (eherrerosj) [reply data/ecosystems/t/ton.toml:2534]: @tsivarev bumping ^

File: data/ecosystems/t/ton.toml [hunk 14]
```diff
@@ -1995,21 +2772,42 @@ url = "https://github.com/moonsharkxyz/twa"
...
 url = "https://github.com/murluki/ton-docs"
 
 [[repo]]
 url = "https://github.com/murluki/twa-ton-eshop"
 
+[[repo]]
+url = "https://github.com/muxe1/crypto-pay-api-sdk"
+
+[[repo]]
+url = "https://github.com/muxe1/proxmuxel"
```
[the remark this answers is not in the document]
Comment (eherrerosj) [reply data/ecosystems/t/ton.toml:2783]: @tsivarev

File: data/ecosystems/t/ton.toml [hunk 15]
```diff
@@ -1029,170 +1275,385 @@ url = "https://github.com/dTelecom/ton"
...
+url = "https://github.com/godemodegame/TonSmartChallenge-1"
+
+[[repo]]
+url = "https://github.com/godemodegame/tonswap-web"
+
+[[repo]]
+url = "https://github.com/goplus-labs/BC-Analyzer"
+
+[[repo]]
+url = "https://github.com/goplus-labs/langchain-tutorials"
```
[the remark this answers is not in the document]
Comment (eherrerosj) [reply data/ecosystems/t/ton.toml]: @tsivarev

**Review by eherrerosj — CHANGES_REQUESTED** [review]

**Review by eherrerosj — APPROVED** [review]
thanks for all the work!

**Pull request merged**